inc/featured-media.php

```diff
@@ -157,109 +132,51 @@ function largo_get_featured_image_hero($post = null, $classes = '') {
 		'the_post' => $the_post
```

Comment:
If we're adding `partials/hero-featured-embed.php` to the list of things that get rendered with this context, then `$featured_media` and `$gallery_ids` need to be passed in this array if that's appropriate to the rendering.